frontend/src/app/corpus-definitions/form/documentation-form/documentation-form.component.html

```diff
@@ -0,0 +1,32 @@
+<h1 class="title">Add documentation</h1>
+<p class="subtitle">Provide information about your corpus</p>
+
+<p class="block">
+    Add documentation to help users understand what your corpus is, and how they can use
+    it in their research. You can use
+    <a href="https://www.markdownguide.org/">markdown</a> to style the content.
```

Comment:
Do we allow any markdown flavour like GitHub? May consider adding link to documentation if so.

Reply:
We render documentation with [marked](https://marked.js.org/). They specify what they support, but it's not really a user-friendly guide. We could link to something like like [markdownguide.org](https://www.markdownguide.org/).

Reply:
Yeah, in that case use the existing link.